src/blocks/EqPolynomial.sol

```diff
@@ -5,53 +5,14 @@ import "src/blocks/pasta/Vesta.sol";
 import "src/blocks/pasta/Pallas.sol";
 
 library EqPolinomialLib {
-    function evaluateVesta(uint256[] memory r, uint256[] memory rx) public pure returns (uint256) {
```

Comment:
Is it worth doing a similar refactor for `evalsPallas` and `evalsVesta`?

Reply:
Agree. One thing to note: this kind of evaluation is not used in e2e verification "pp-spartan" flow which is default - as far as I remember it was used in "regular" spartan. I guess it is a normal situation when we have building block which is not used currently but either used previously or may be can be used in future.